zebrad/src/application.rs

```diff
@@ -398,6 +398,11 @@ impl Application for ZebradApp {
         }
         components.push(Box::new(Tracing::new(tracing_config)?));
 
+        // Log git metadata and platform info when zebrad starts up
+        if is_server {
+            tracing::info!("{}", metadata_section);
```

Comment:
Optional: This happens early on startup, so it might help to give users a bit more of a clue why we're printing this.

```suggestion
            tracing::info!("Diagnostic {}", metadata_section);
```